Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -232,7 +232,7 @@
     <string name="onboarding_privacy_body">@string/lorem_ipsum</string>
     <string name="onboarding_privacy_body_emphasized">@string/lorem_ipsum</string>
     <!-- 3 Permission -->
-    <string name="onboarding_tracing_headline">Wie Sie die Risikoerkennung ermöglichen</string>
+    <string name="onboarding_tracing_headline">Wie Sie die Risiko-Ermittlung ermöglichen</string>
```

Comment:
This is the only occurrence of "Risikoerkennung". Went for "Risiko-Ermittlung" in line with #42

Reply:
For line 244, I suggest to go for "Sie wurden positiv getestet", so change present tense to past tense.